### PR TITLE
Update Java version for IHS & WAS.  Update DB2 version for GitWorkflow

### PR DESCRIPTION
**Pull request by martinfanning1**
Update Java version for IHS & WAS.  
Update DB2 version for Github Workflow test

Molecule test for WAS & IHS passed with new Java Version

### CODE REVIEW

**Comment by martinfanning1** [comment]
So to update further on the failing db2 test..  I think i know why and what is happening..

**TLDR:** Issue is with Rocky and DB2 11.5.9.0 fixpack

There are 2 issues ultimately.

**_Issue 1 - missing library_**
The Github Workflow error suggests there is a missing library
` DBT3514W  The db2prereqcheck utility failed to find the following 32-bit library file: "libstdc++.so.6".`

From some research this is a required library for DB2 - https://www.ibm.com/docs/en/db2/11.5?topic=servers-linux
`libstdc++.so.6 is required for Db2 database servers and clients.`

When the Github Workflow runs i can see these libs installed
defaultworkflow

I had earlier in the week installed DB2 11.5.9.0 on a VM with RHEL and this passed the DB2 installation part- [Jenkins Output](https://spmdevops-team-apollo.jenkins.commops.merative.com/job/DevOps-Jenkins-ProvisionerWrapper/604/console)

So to test out the missing library - i modified the DB2 molecule test to install IIM also - similar to the VM deployment flow.
This looked to have added the required library needed
iimworkflow

However the molecule tested failed albeit with a different error this time..  This leads us onto issue 2

**_Issue 2 - DB2 11.5.9.0 changes and impact on Rocky_**|
From googling furter on issues with DB2 11.5.9.0 i found the following issues highlighted for deploying the fixpack on Rocky - 
https://dba.stackexchange.com/questions/333207/db2-v11-5-9-0-installation-error-while-loading-shared-libraries-libaws-cpp-sdk

It appears to stem from new changes introduced in the fixpack
Here is a list of these files in 11.5.8.0


And the same in 11.5.9.0


This matches what is explained in the link above..

So to test out this theory - while running the molecule test - i created the symlinks on the fly when the DB2 installer had laid them on the filesystem.

The install passed the point of the previous failure
Screenshot 2024-04-12 at 18 10 09

There are failures further on with the creation of the database - but i also had these locally with DB2 11.5.8.0.. Log attached for this failure.

So ultimately to get the DB2 11.5.9.0 molecule test to pass via Github workflow i think we need to implement the above 2 changes..

[1159_molecule.txt](https://github.com/merative/spm-middleware/files/14961524/1159_molecule.txt)

**Comment by martinfanning1** [comment]
Closing this PR

have separated it out to allow progression with Java updates for IHS and WAS

https://github.com/merative/spm-middleware/pull/80 - Java update for IHS & WAS
https://github.com/merative/spm-middleware/pull/81 - DB2 Molecule Test
